packages/connect/README.md

```diff
@@ -1 +1,51 @@
+@0xproject/connect
+------
+
 This repository contains a Javascript library that makes it easy to interact with Relayers that conform to the [Standard Relayer API](https://github.com/0xProject/standard-relayer-api)
+
+## Installation
+
+```bash
+yarn add @0xproject/connect
+```
+
+## Usage
+
+* [Docs](https://0xproject.com/docs/connect)
+* [Tutorials](https://0xproject.com/wiki#connect)
+
+## Contributing
+
+We strongly encourage our community members to help us make improvements and to determine the future direction of the protocol. To report bugs within this package, please create an issue in this repository. 
+
+[CONTRIBUTING.md](../../CONTRIBUTING.md)
```

Comment:
`Please read our [contribution guidelines](../../CONTRIBUTING.md) before getting started.`

Reply:
I updated the template too. Mind changing this in all of them?